peers/src/error.rs

```diff
@@ -38,8 +38,6 @@ pub enum Error {
     RequestTooLarge,
     /// gRPC failure: {0}
     Grpc(GrpcError),
-    /// Internal retry failure: {0}
-    RetryInternal(String),
```

Comment:
I would hope that only nodes are looking at this error type so don't think this change has an impact outside the peers.

Reply:
closing discussion, no complaints